Integrations/integration-Cisco-umbrella.yml

```diff
@@ -31,495 +31,1834 @@ configuration:
+        'ks_test': { 'Name': 'Kolmogorov-Smirnov Test', 'ContextKey': 'KolmogorovSmirnovTest', 'Info': "Kolmogorov-Smirnov test on geodiversity. 0 means that the client traffic matches what is expected for this TLD" },
+        'attack': { 'Name': 'Attack Name', 'ContextKey': 'AttackName', 'Info': 'The name of any known attacks associated with this domain. Returns blank if no known threat associated with domain' },
+        'threat_type': { 'Name': 'Threat Type', 'ContextKey': 'ThreatType','Info': 'The type of the known attack, such as botnet or APT. Returns blank if no known threat associated with domain' }
     }
 
-    // used to describe result on getDomainDNSHistory function
-    var IpDnsFeatureInfo = {
-        rr_count: 'Number of records of that type mapping to the given IP',
-        ld2_count: 'Number of 2-level names mapping to the given IP',
-        ld3_count: 'Number of 3-level names mapping to the given IP',
-        ld2_1_count: 'Number of 2-level names, without the TLD, mapping to the given IP',
-        ld2_2_count: 'Number of 3-level names, without the TLD, mapping to a given IP',
-        div_ld2: 'ld2_count divided by the number of records',
-        div_ld3: 'ld3_count divided by the number of records',
-        div_ld2_1: 'ld2_1_count divided by the number of records',
-        div_ld2_2: 'ld2_2_count divided by the number of record'
+    #used to describe result on getDomainDNSHistory function
+    IP_DNS_FEATURE_INFO = {
+        'rr_count': 'Number of records of that type mapping to the given IP',
+        'ld2_count': 'Number of 2-level names mapping to the given IP',
+        'ld3_count': 'Number of 3-level names mapping to the given IP',
+        'ld2_1_count': 'Number of 2-level names, without the TLD, mapping to the given IP',
+        'ld2_2_count': 'Number of 3-level names, without the TLD, mapping to a given IP',
+        'div_ld2': 'ld2_count divided by the number of records',
+        'div_ld3': 'ld3_count divided by the number of records',
+        'div_ld2_1': 'ld2_1_count divided by the number of records',
+        'div_ld2_2': 'ld2_2_count divided by the number of record'
     }
 
-    var extractDomainName = function(url) {
-        var hostname;
-        //find & remove protocol (http, ftp, etc.) and get the hostname
-        if (url.indexOf("://") > -1) {
-            hostname = url.split('/')[2];
-        }
-        else {
-            hostname = url ? url.split('/')[0] : '';
-        }
-
-        //find & remove port number
-        hostname = hostname.split(':')[0];
-
-        return hostname;
-    }
+    ''' HELPER FUNCTIONS '''
+
+    def extract_domain_name(url):
+        # Remove protocol, removes directory after the host name, returns only the host name.
+        return url.split("//")[-1].split("/")[0]
+
+
+    def http_request( api_endpoint, params_dict=None, method='GET',data_list=None):
```

Comment:
why don't we use this function in commonServer as well? all integrations seem to have it

Reply:
we could/should